migrations/V161__Add_TRB_Admin_Note_Documents_Link_Table.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE trb_admin_notes_trb_request_documents_links (
+    id UUID PRIMARY KEY NOT NULL,
+    trb_request_id uuid NOT NULL REFERENCES trb_request(id),
```

Comment:
I jumped on a call with @StevenWadeOddball  to review this and it took us a while to understand exactly what was going on here. How do you feel about adding a `COMMENT` to the request ID column to let future devs know this field is in place purely for the restraints below to make sure both notes and docs share the same request ID?

Reply:
I can definitely do that.

Reply:
Added (along with some other `COMMENT`s) in https://github.com/CMSgov/easi-app/pull/2254/commits/0c845572f0f53c828ee9764303bda82c3815e895 - let me know how that reads, if it's easy to understand.